resources/Dockerfile.skill

```diff
@@ -4,33 +4,24 @@ FROM ros:jazzy AS base
 WORKDIR /opt/ros/underlay
 
 ENV ROS_HOME=/tmp
-ENV RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+ENV RMW_IMPLEMENTATION=rmw_zenoh_cpp
 
 # underlay stage: base + dependencies built
 FROM base AS underlay
 
 ADD src/intrinsic_sdk_ros /opt/ros/underlay/src/intrinsic_sdk_ros
-
-RUN git clone https://github.com/eclipse-zenoh/zenoh-plugin-dds \
-    -b release-1.0.2 \
-    --recurse-submodules \
-    /opt/ros/underlay/src/zenoh-plugin-dds
+# TODO(Yadunund): Switch to ZENOH_CONFIG_OVERRIDE after the next jazzy sync.
+COPY zenoh_config.json5 /opt/zenoh_config.json5
 
 RUN . /opt/ros/jazzy/setup.sh \
     && apt-get update \
-    && apt install -y clang-14 ros-jazzy-ament-cmake-vendor-package cargo \
-    && colcon build \
-        --continue-on-error \
-        --cmake-args -DCMAKE_C_COMPILER=clang-14 -DCMAKE_CXX_COMPILER=clang++-14 \
-        --event-handlers=console_direct+ \
-        --merge-install \
-        --packages-up-to zenoh_bridge_dds \
+    && apt install -y clang-16 ros-jazzy-ament-cmake-vendor-package cargo \
     && colcon build \
+        --cmake-args -DCMAKE_C_COMPILER=clang-16 -DCMAKE_CXX_COMPILER=clang++-16 \
         --continue-on-error \
         --event-handlers=console_direct+ \
         --merge-install \
-        --packages-up-to intrinsic_sdk \
-        --packages-skip zenoh_bridge_dds
+        --packages-up-to intrinsic_sdk
```

Comment:
Same comment here, I don't think we need clang-16 or to override the cxx compiler here.

Reply:
iirc I was not able to build the sdk without clang-16 on `main`. Do we no longer need to do that with this branch?

Reply:
I don't use clang at all on my branch at all.

Reply:
This is the equivalent dockerfile I use to build this repository into an image: https://github.com/intrinsic-dev/intrinsic_sdk_ros/pull/19/files#diff-1e729070fdf6df69664071660444803bcdb8df9ec3e1b1551276006dad804a8d

And it builds the sdk as well as zenoh bridge (which itself uses cargo) with any issues. zenoh bridge brings cargo in automatically via rosdep install, so I don't explicitly install it either.

Reply:
Dropped it https://github.com/intrinsic-dev/intrinsic_sdk_ros/pull/20/commits/aa875c72e8cff5e79f93f511645c06aba9b64cce